src/quantum/tp6.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import _ from '../utils/more_lodash';
+import { TP6_PAGES } from './config';
+import DashboardPage from '../components/DashboardPage/index';
+import PerfherderGraphContainer from '../containers/PerfherderGraphContainer/index';
+
+
+const styles = {
+  // This div helps with canvas size changes
+  // https://www.chartjs.org/docs/latest/general/responsive.html#important-note
+  body: {
+    backgroundColor: 'white',
+  },
+  area: {
+    display: 'flex',
+  },
+  column: {
+    flex: '50%',
+    alignItems: 'center',
+  },
+  chart: {
+    justifyContent: 'center',
+    padding: '1rem',
+  },
+};
+
+class TP6 extends React.Component {
+  constructor(props) {
+    super(props);
+    const { classes, match } = this.props;
+    const bits = match.params.bits;
+
+    this.state = {
+      allCharts: _
+        .chain(TP6_PAGES)
+        .filter(row => row.bits === bits)
+        // GROUP BY title
+        .groupBy(row => row.title)
+        // LOOP OVER EACH KEY/VALUE
+        .toPairs()
+        .map(([title, series], i) => (
+          <div key={`page_${bits}_${i}`} className={classes.chart}>
+            <PerfherderGraphContainer
+              title={title}
+              series={_
+                .chain(series)
+                .map((s) => {
+                  return {
+                    label: s.label,
+                    seriesConfig: s,
+                  };
+                })
+                .value()
+              }
+            />
+          </div>
+        ))
+        // SPLIT INTO TWO
+        .chunk(2)
+        // TRANSPOSE, SO WE HAVE TWO COLUMNS
+        .unzip()
+        .value(),
+    };
+  }
+
+  render() {
+    const { allCharts } = this.state;
+    const { classes, match } = this.props;
+    const bits = match.params.bits;
+
+    return (
+      <div className={classes.body}>
+        <DashboardPage key={bits} title={`TP6 - Page load on ${bits} bits`}>
+          <div className={classes.area}>
+            <div className={classes.column}>
```

Comment:
To make your life easier please do one column for now.
People will be fine if we don't take advantage of the whole width.
We can visit `display: flex` later. We might want to use CSS grids.
You can also chat with hassan/sclements if you want help with this.

You can do the following once you fix the state:
```javascript
{allCharts.map(([title, series], i) => (
   <div key={`page_${bits}_${i}`} className={classes.chart}>
     <PerfherderGraphContainer
        title={title}
        series={series.map({ label, seriesConfig } => ({ label, seriesConfig })))}
      />
  </div>
)}
```

Reply:
@armenzg Are you suggesting we sacrifice visual style for better coding style? I believe layout is complicated because of the miserable state of CSS and layout on the web**.  I think we should keep the two columns; if only to be held up as an example to be replaced by something better later.  I have no problem moving to CSS Grid, but the code will not be any simpler.

@helfi92 @sarah-clements  Do you have suggestions?  One chart per "row" seems too sparse, and probably ugly.  It would be nice if there was a CSS element that can choose the right number of columns, and fill them, but I suspect a CSS solution would be inscrutable.

** I make a distinction between "layout" (the inter-component positioning of UI elements) to "styling" (a component's look). CSS has been terrible at layout, and it continues to change; whatever we use to layout the page will probably change next year.  Specifically, the years of CSS advocates complaining that `<table>` should not be used for layout, and then to have CSS grids is comical.  Comical, not just because it is the same thing by another name, but still oblivious to how layout should be done.  When the standards committees finally use what building architects and machinists have been using to position elements for centuries (software implementations are called C.A.D) then we will have a stable, and simpler, specification for layout.

Reply:
@helfi92: This looks interesting, and I will try it.

Reply:
@helfi92 please take a look at what I did with Grid:  It is fine, but I would have hoped it would choose the number of columns based on the window width, and not the physical form factor.  Maybe you know more.